python/cuml/neighbors/nearest_neighbors.pyx

```diff
@@ -922,7 +907,7 @@ def kneighbors_graph(X=None, n_neighbors=5, mode='connectivity', verbose=False,
         metric=metric,
         p=p,
         metric_params=metric_params,
-        output_type=output_type,
+        output_type=cuml.global_settings.root_cm.output_type
```

Comment:
I'm open to suggestion on this one. There might be a better way to get the output type.

Reply:
I think if the user uses a context manager to set this, it would be set that way so you don’t need to pass it here (you might need to do some testing to see if I’m correct)

Reply:
If I delete this line,  `test_nearest_neighbors.py::test_knn_graph` would fail because the `output_type` is replaced to `mirror` by the `api_return_sparse_array` function decorator, and `NearestNeighbors` cannot be initialized with `output_type==mirror`